packages/components/src/Badge/Badge.component.js

```diff
@@ -54,7 +66,9 @@ Badge.propTypes = {
 	label: PropTypes.string,
 	category: PropTypes.string,
 	onDelete: PropTypes.func,
+	deleteId: PropTypes.string,
```

Comment:
Please avoid having multiple ids as input. You can align it with other components we have : 
* get a unique id from props
* if present, generate a delete id and a select id from it, and set it

Reply:
+ add a unit test please

Reply:
@jsomsanith thanks, updating..